ihatemoney/tests/api_test.py

```diff
@@ -577,19 +577,19 @@ def test_bills_with_calculation(self):
     def test_currencies(self):
         # check /currencies for list of supported currencies
         resp = self.client.get("/api/currencies")
-        self.assertTrue(201, resp.status_code)
+        self.assertEquals(200, resp.status_code)
```

Comment:
I supposed that `200` is really what's wanted here, instead of `200`, as this is not an object creation.

Reply:
Yes, it's supposed to be 200.